.gitignore

```diff
@@ -26,6 +26,7 @@ builder/rootfs/usr/bin
 cache/image/bin/
 client/dist/
 client/makeself/
+client-go/deis
```

Comment:
In the end it should replace the `client/` dir IMHO, but it would be safest to be able to make a python client build up until that point. I think going back in history with `git` is sufficient to allow that, or else the old client code could move to client-old or somesuch. tl:dr, however you want to do it, but when we find an unpexected issue with the go CLI we will be glad we still have the python CLI working as a fallback.